.github/workflows/check_changelog.yml

```diff
@@ -17,6 +17,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: scientific-python/action-towncrier-changelog@1d7332022f76e36fe8ce2d716b851f3f98063c62  # v1.0.0
+      if: github.repository == 'astropy/astropy'
```

Comment:
Maybe move it higher up.

https://github.com/astropy/astropy/blob/main/.github/workflows/ci_cron_weekly.yml

Reply:
Will that make the next step fail since it relies on the passing of `initial_checks`